docs/measurement/schema/location.md

```diff
@@ -176,3 +177,9 @@ both of the following queries would match `amazon technologies inc.` network
 { "type": "magic", "value": "aws" },
 { "type": "magic", "value": "amazon" }
 ```
+
+magic queries can be combined. The following query will match `London, Ohio, USA`
+
+```json
+{ "type": "magic", "value": "london+ohio+usa" }
```

Comment:
In our case it wont match anything since we dont have a probe in London in Ohio.
Maybe a clearer example that would work on our existing probes? E.g. "google+belgium" to return results from Google Cloud probes only in Belgium.

Reply:
I wanted to show you can form a very precise queries, but I guess this exmaple would be close enough.

Reply:
I understand but I want our examples to be copy-paste functioning :)